.github/workflows/check_vue_tsc.yml

```diff
@@ -0,0 +1,20 @@
+name: check-vue-tsc
+on: 
+  pull_request
+
+jobs:
+  vue-tsc:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 7
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: pnpm
+      - name: Install vue-tsc
+        run: pnpm i vue-tsc -D
```

Comment:
@sonya0504 We already have the `vue-tsc` package defined in `package.json`. I think we can simply install the package using `pnpm i`.

Reply:
Sure, I changed